main.rb

```diff
@@ -118,149 +129,27 @@ def my_map(is_proc = nil)
   end
 
   def my_inject(temp_option = nil)
-    new_items = temp_option || shift
-    my_each do |item|
-      new_items = yield(new_items, item)
+    if is_a? Range
+      min = self.min
+      max = self.max
+      new_items = temp_option || 0
+      while min <= max
+        new_items = yield(new_items, min)
+        min += 1
+      end
+    else
+      new_items = temp_option || shift
+      my_each do |item|
+        new_items = yield(new_items, item)
+      end
```

Comment:
#### :bug: 

- `my_inject` should be able to accept a `Symbol` as an operator (e.g `:+`, `:*`) when no block is given:
   ```ruby
   irb(main):055:0> [1,2,3].inject(:+)
   => 6
   irb(main):056:0> [1,2,3].my_inject(:+)
   LocalJumpError  (no block given (yield))
   ```

- `my_inject` should be able to accept two arguments, first as an initial value and the other as a `Symbol`:

   ```ruby
   irb(main):057:0> [1,2,3].inject(5,:+)
   => 11
   irb(main):058:0> [1,2,3].my_inject(5,:+)
   ArgumentError (wrong number of arguments (given 2, expected 0))
   ```

### Find more details about [`inject?` here](https://ruby-doc.org/core-2.6.4/Enumerable.html#method-i-inject)